### ISSUE TITLE
npm dependencies conflict in template

### ISSUE THREAD

**Issue by epogrebnyak** [gethinode/hinode#901]
## Describe the bug

In https://github.com/gethinode/template `npm install` results in

```
npm ERR! While resolving: eslint-config-standard@17.1.0
npm ERR! Found: eslint-plugin-n@17.3.1
npm ERR! node_modules/eslint-plugin-n
npm ERR!   dev eslint-plugin-n@"^17.3.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-n@"^15.0.0 || ^16.0.0 " from eslint-config-standard@17.1.0
npm ERR! node_modules/eslint-config-standard
npm ERR!   dev eslint-config-standard@"^17.1.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: eslint-plugin-n@16.6.2
npm ERR! node_modules/eslint-plugin-n
npm ERR!   peer eslint-plugin-n@"^15.0.0 || ^16.0.0 " from eslint-config-standard@17.1.0
npm ERR!   node_modules/eslint-config-standard
npm ERR!     dev eslint-config-standard@"^17.1.0" from the root project
```

**Comment by markdumay** [comment]
Thanks for reporting the issue, however, I cannot reproduce it. When submitting a bug in the future, would you mind filling in the template? It includes a section on how to reproduce the error and it clarifies the host environment.

Regarding the submitted error logs, the `eslint-config-standard` package is not compatible with the latest `eslint-plugin-n` yet. See https://github.com/standard/eslint-config-standard/issues/410 for the latest status. For now, the Hinode (template) repository uses `eslint-plugin-n` `v16.6.2` instead (see PR https://github.com/gethinode/template/pull/311 for an automated test). 

I suspect you upgraded the npm modules locally (e.g. `npx npm-check-updates`) and ran into this issue when running `npm install`? My advice would be to use the current `package.json` file as is.

**Comment by epogrebnyak** [comment]
Thank you for prompt response. There where not many steps involved rather than following the tutorial to create the repo from template and run `npm install`. I did not change `package.json` (I think). I could get past the issue with `npm install --force`. I can try repoduce the error once more and report it back here.

**Comment by ammakr** [comment]
I got the same issue on debian 12. is there any way to handle it in a clean way, without using --force?

**Comment by ammakr** [comment]
I found the solution, just change one of the dep version:    ` "eslint-plugin-n": "^16.0.0",`

**Comment by markdumay** [comment]
@epogrebnyak, which host OS are you using? I cannot reproduce the error on macOS or Ubuntu (via GitHub codespaces), so perhaps it's tied to a specific host environment. I'm fine to adjust the base repositories, however, I'd like to understand the cause of the issue a bit better.

**Comment by epogrebnyak** [comment]
Should have said i'm on Windows

**Comment by ammakr** [comment]
> Thanks! At `package.json` is that?  

Yep, and it is not some OS related problem, i happen to come across this on debian 12.

**Comment by markdumay** [comment]
The issue should be solved now (see [v0.14.0](https://github.com/gethinode/template/releases/tag/v0.14.0) for details). PR https://github.com/gethinode/template/pull/311 tracks the progress of `eslint-plugin-n`.
